toolchain/check/testdata/packages/explicit_imports.carbon

```diff
@@ -0,0 +1,35 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+
+// --- api.carbon
+
+package Api api;
+
+// --- api_lib.carbon
+
+package Api library "lib" api;
+
+// --- import_api.carbon
+
+import Api;
+import Api library "lib";
+
+// CHECK:STDOUT: file "api.carbon" {
+// CHECK:STDOUT: }
```

Comment:
There's already a TODO for this in the formatter. Since it would touch every semir test, and may be subject to preferences about format, I'd rather split it out.

Reply:
Yes, please don't block this PR on this.